src/hotspot/cpu/riscv/c2_MacroAssembler_riscv.cpp

```diff
@@ -1700,14 +1700,14 @@ void C2_MacroAssembler::compress_bits_v(Register dst, Register src, Register mas
   vmv_v_i(v8, 0);
   // load the mask data(in bits).
   vsetivli(x0, 1, sew, lmul);
```

Comment:
Please also change line into "vsetivli(x0, 1, sew, Assembler::m1)" for consistency. Otherwise LGTM.

Reply:
Modified.